databases/heroku-pgbouncer-buildpack/index.js

```diff
@@ -14,8 +14,6 @@ const app = express()
 const port = process.env.PORT || 3000
 
 app.get('/', async (req, res) => {
-  await clientWithQueryStringParam.user.deleteMany({})
```

Comment:
Why remove?

Reply:
Because it's deleting all users, and we can make this isolated across the runs and only create / delete one user.

Same answer to the other comments

Reply:
Hm ok, I think this pattern is used in many other places  but might make sense to universally change.

Reply:
Did a revert, better to do in a separate PR